src/legacy/ui/public/vislib/lib/dispatch.js

```diff
@@ -75,10 +75,13 @@ export function VislibLibDispatchProvider(Private, config) {
      * @param d {Object} Data point
      * @returns event with list of data points related to the click
      */
-    clickEventResponse(d) {
-      const _data = d3.event.target.nearestViewportElement ?
-        d3.event.target.nearestViewportElement.__data__ : d3.event.target.__data__;
-      const isSlices = !!(_data && _data.slices);
+    clickEventResponse(d, props = {}) {
+      let isSlices = props.isSlices;
```

Comment:
Is this just for testing to avoid mocking d3 here?

Reply:
yes ... well it would be nice if we wouldn't use that d3 magic at all, but removing it would take much more time i guess